pkg/controller/operators/catalog/operator.go

```diff
@@ -1160,6 +1160,9 @@ func (o *Operator) ExecutePlan(plan *v1alpha1.InstallPlan) error {
 				_, err = o.OpClient.KubernetesInterface().RbacV1().ClusterRoles().Create(&cr)
 				if k8serrors.IsAlreadyExists(err) {
 					// if we're updating, point owner to the newest csv
+					if cr.ObjectMeta.Labels == nil {
+						cr.ObjectMeta.Labels = map[string]string{}
+					}
 					cr.Labels[ownerutil.OwnerKey] = step.Resolving
```

Comment:
This line also needs to be changed to:
```suggestion
					cr.ObjectMeta.Labels[ownerutil.OwnerKey] = step.Resolving
```

Reply:
These are the same thing (`ObjectMeta` is embedded in the `ClusterRole`) but I will commit this change so that it matches the other PR.